parts/linux/cloud-init/artifacts/setup-custom-search-domains.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -x
 source {{GetCSEHelpersScriptFilepath}}
+source {{GetCSEHelpersScriptDistroFilepath}}
```

Comment:
I see apt-get on line 10, meaning we should also make this file distro-sensitive

Reply:
As we talked offline, let's get rid of this setup-custom-search-domains.sh. It's not needed